src/main/java/com/rcv/ContestConfig.java

```diff
@@ -128,10 +128,17 @@ static ContestConfig loadContestConfig(String configPath) {
       Logger.log(Level.INFO, "Successfully loaded contest config: %s", configPath);
       // perform some additional sanity checks
       if (rawConfig.validate()) {
-        // checks passed so continue processing
-        config = loadContestConfig(rawConfig, new File(configPath).getParent());
+        // source folder will be the parent of configPath
+        String parentFolder = new File(configPath).getParent();
+        // if there is no parent folder use current working directory
+        if (parentFolder == null) {
+          parentFolder = System.getProperty("user.dir");
+        }
+        config = loadContestConfig(rawConfig, parentFolder);
       } else {
-        Logger.log(Level.SEVERE, "Failed to create contest config!");
+        Logger.log(Level.SEVERE, "Failed to create raw contest config!");
```

Comment:
Please combine all log messages like this into one and use line breaks /n when necessary. Some log parsers will count this as 3 severe errors, for example. There are some other cases I see with 2-3 messages in succession elsewhere, so please combine those too.

Reply:
that's a good point, will do